extensions/amp-timeago/amp-timeago.md

```diff
@@ -46,10 +48,15 @@ Provides fuzzy timestamps by formatting dates as `*** time ago` (for example, 3
 
 ## Behavior
 
-Provides fuzzy timestamps that you can use on your AMP pages. This component is based on <a href="https://github.com/hustcc/timeago.js">timeago.js</a>.
+Use the amp-timago component to count up to, or away from, a specified date and time.
+
+The component replaces the text node with a fuzzy timestamp, such as `in 30 years` or `3 hours ago`.
+If using the `cutoff` attribute, and time has past the cutoff, the text node will display the specified date in the `datetime` attribute.
```

Comment:
I'm still confused about this component... I just tried it in the playground, and the validator enforces the presence of the `datetime` attribute. It looks like the component always use the timestamp specified in the `datetime` attribute, so I'm not sure why there needs to be a text node at all.

If I remove the text node completely, I get this error:

```
Content cannot be empty. Found in: amp-timeago
```

But it looks like I can put anything into the text node and anything seems to work, including just a space. The text node seems completely ignored, yet the component complains if I don't specify a text node.

Reply:
Good point. Might be worth filing an issue in the amphtml repo. @edhollinghurst originally implemented the extension.

Reply:
I've added a bit about requiring a character in the text node, and will be merging and closing the issue, as this is no longer a documentation issue.